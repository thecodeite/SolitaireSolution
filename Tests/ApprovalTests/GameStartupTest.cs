using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;
using Solitaire.Models;

namespace Tests.ApprovalTests
{
    [TestFixture]
    [UseReporter(typeof(DiffReporter))]
    class GameStartupTest
    {
        [Test]
        public void the_board_should_appear_correct_on_startup()
        {
            // When the game is started
            var game = new Game();

            // the board should be correct
            string board = game.Render();
            Approvals.Verify(board);
        }
    }
}
