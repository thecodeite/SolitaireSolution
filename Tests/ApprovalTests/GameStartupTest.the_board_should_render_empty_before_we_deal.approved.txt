 ColumnNames   S[T]ack        [1] [2] [3] [4] [5] [6] [7] [D] [H] [C] [S]
-------------------------------------------------------------------------
                                                                         
                                                                         
                                                                         
                                                                         
                                                                         
                                                                         
                                                                         
